### DISCUSSION TITLE
Loction services with itlwm.kext + heliport possible?

### DISCUSSION THREAD

**Discussion by franzasensi** [daliansky/XiaoMi-Pro#763] [Q&A]
Is there any way to activate location services when using itlwm.kext + heliport??

Also, will it be an airportitlwm.kext that works with Sequoia anytime soon?

Thanks!

**Comment by stevezz3** [comment]
That's one of the drawback of itlwm+HeliPort. It spoofs our Intel wireless card as an ethernet card, so the location service is not likely to be supported.

The Sequoia version of AirportItlwm needs unknown time (my guess is a couple of months) to develop. That's also the reason why I don't have much passion to add support for Sequoia.

You can follow https://github.com/OpenIntelWireless/itlwm/issues for the updates of the driver.
